deployment/aks-periscope.yaml

```diff
@@ -28,6 +28,9 @@ rules:
 - apiGroups: ["apiextensions.k8s.io"]
   resources: ["customresourcedefinitions"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["access.smi-spec.io", "specs.smi-spec.io", "split.smi-spec.io"]
```

Comment:
This file is deprecated and shall not be updated anymore. These changes should be added to the corresponding kustomize resource file instead.
@Tatsinnit do you agree?

Reply:
Hi @arnaud-tincelin, I think this should be resolved now that I've edited the corresponding kustomize resource file: https://github.com/Azure/aks-periscope/pull/71/files#r671532572